userfiles/modules/users/terms/index.php

```diff
@@ -5,14 +5,14 @@
 $terms_label_cleared = mb_trim($terms_label_cleared);
 
 if ($terms_label_cleared == '') {
-    $terms_label = 'I agree with <a href="' . site_url() . 'terms" target="_blank">terms and conditions</a>';
+    $terms_label = 'I agree with the <a href="' . site_url() . 'terms" target="_blank">Terms and Conditions</a>';
 }
 ?>
-<div class="row" style="margin-top: 20px;margin-bottom: 20px;">
+<div class="row" style="margin-top: 10px;margin-bottom: 10px;">
     <div class="col-xs-12" class="i_agree_with_terms_row_popup">
         <div class="form-group">
             <div class="custom-control custom-checkbox">
-                <h5><?php _e('You must agree to terms and conditions to continue'); ?></h5>
+                <!-- <h5><?php _e('You must agree to the Terms and Conditions to continue'); ?></h5> //-->
```

Comment:
this must be not commented out

Reply:
Might be nice to have a configurable option to also show a newsletter subscribe checkbox similar to the example [here](https://www.formstack.com/templates/gdpr-compliant-contact-form), and allow the default text to be edited (similar to the terms consent text)?